### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by rtonata88**
In this PR I have added styling with bootstrap framework.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,25 +6,85 @@
     <title>Leaderboard</title>
```
Comment (GzimAsani) [inline src/index.html:6]: - [ ] In your readme file the live link does not represent your current project. Kindly, update  it!

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @rtonata88                                 
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 
- Good job with the implementation :+1: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (rtonata88) [reply src/index.html:6]: I build the project, the dist has now updated.

**Review by leolpaz — APPROVED** [review]
Hi @rtonata88 , 

Your project is complete!
Congratulations! 🎉🎉 

To highlight:

- Congratulations on implementing the requirements

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**